calico_kubernetes/calico_kubernetes.py

```diff
@@ -53,12 +55,47 @@ def delete(self, pod_name, docker_id):
         self.pod_name = pod_name
         self.docker_id = docker_id
 
+        if not self._datastore_client.profile_exists(self.pod_name):
+            print "A profile with name %s does not exist" % self.pod_name
+            sys.exit(1)
```

Comment:
We don't want to sys.exit here.

I think we should be trying to delete the profile, catching the exception if it doesn't exist, logging out, and then continuing.